ui/raidboss/data/04-sb/raid/o2s.js

```diff
@@ -415,8 +423,10 @@
         'Main Quake': '本震',
         'Maniacal Probe': '触手乱舞',
         'Paranormal Wave': '呪詛波',
+        'Probes': 'Probes', // FIXME
         'Tremblor': '地震',
-        'Unknown Ability': 'Unknown Ability',
+        'Unknown Ability': 'Unknown Ability', // FIXME
```

Comment:
:thinking:

Reply:
this is just a relic of old times...
i just starded with the existing timelines and this line was part of it :x

Reply:
removed this old relik